docker/circleci/run_pytests.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -e
+set -x
+set -u
+
+PYTHON_VERSION=$( python -c "import sys; print('{}{}'.format(sys.version_info[0], sys.version_info[1]))" )
+
+# Create necessary directories
+mkdir -p /scratch/pytest /scratch/crashfiles /scratch/logs/py${PYTHON_VERSION}
+
+# Create a nipype config file
+mkdir -p /root/.nipype
+echo '[logging]' > /root/.nipype/nipype.cfg
+echo 'log_to_file = true' >> /root/.nipype/nipype.cfg
+echo "log_directory = /scratch/logs/py${PYTHON_VERSION}" >> /root/.nipype/nipype.cfg
+
+# Enable profile_runtime tests only for python 2.7
+if [[ "${PYTHON_VERSION}" -lt "30" ]]; then
+    echo '[execution]' >> /root/.nipype/nipype.cfg
+    echo 'profile_runtime = true' >> /root/.nipype/nipype.cfg
+fi
+
+# Run tests using pytest
+# NOTE_dj: this has to be improved, help needed.
+# NOTE_dj: pip install can be probably moved to dockerfiles
+# NOTE_dj: not sure about --xunit-file part (not using with pytest for now)
+cd /root/src/nipype/
+make clean
+pip install -U pytest
+pip install pytest-raisesregexp
+pip install pytest-cov
+pip install click
```

Comment:
`click` is a nipype requirement. and so should `pytest` now.  these should be updated in `nipype/info.py` and `*requirements.txt`